geomstats/geometry/product_riemannian_metric.py

```diff
@@ -3,14 +3,14 @@
 Define the metric of a product manifold endowed with a product metric.
 """
 
+from joblib import delayed, Parallel
```

Comment:
Avoid relative imports

Reply:
Relative imports refer to things like `from . import sibling` though. But maybe a possible rule-of-thumb could be that if one uses more than, say, 2 objects from a module, one should import the toplevel module itself rather than individual objects.